src/test/lib/operators/aggregate_test.cpp

```diff
@@ -25,6 +25,14 @@
 
 namespace hyrise {
 
+class OperatorsAggregateHashTest : public BaseTest {};
+
+TEST_F(OperatorsAggregateHashTest, EmptyHash) {
+  const auto expected_empty_hash = 0;
```

Comment:
why declare and not `EXPECT_EQ(..., 0)`?

Reply:
I wanted to emphasize that both hashes below equal the same value and are not just coincidentally 0 - would you still prefer the inlined 0?

Reply:
If you ask me like this, I would prefer the inlined `0` with a comment why both values are not just coincidentally `0`:D

Reply:
See the comment I added - I though it made sense, but if you say it truly is a coincidence, I can also remove it again :D